gamestonk_terminal/menu.py

```diff
@@ -1,17 +1,21 @@
+import logging
 import os
 
 from matplotlib import pyplot
 from prompt_toolkit import PromptSession
 from prompt_toolkit.eventloop.inputhook import set_eventloop_with_inputhook
 from prompt_toolkit.history import FileHistory
 
+logger = logging.getLogger(__name__)
+
 
 def inputhook(inputhook_context):
     while not inputhook_context.input_is_ready():
         try:
             pyplot.pause(0.1)
         # pylint: disable=unused-variable
```

Comment:
This probably can be removed as the variable `exp`  is now used

Reply:
> The f-strings are more modern and are the way strings are formatted across the project. This is the only styling change request I have.

https://pycodequ.al/docs/pylint-messages/w1203-logging-fstring-interpolation.html

Logging should be done without f-strings.

> I would suggest lowering the default logging level to info to prevent overwhelming of the logs folder for active users. We don't yet have "log rotation" or log removal over time implemented here, so this might be a better default choice.

Default logging goes into a file, the user does not see logging output.

Reply:
> This probably can be removed as the variable `exp` is now used

Fixed

Reply:
Very interesting. Thought you did it from force of habit or so... The reason is for backward compatibility and performance apparently, https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles.

The only other thing I'd say is should we let the user know through a message about the logging? We can couple this to that condition when the `logs` folder is created. Basically saying that from now on we will have logs so we can improve the usage of the terminal, and they can change the logging level allowed in the `config_terminal`.

Reply:
> Very interesting. Thought you did it from force of habit or so... The reason is for backward compatibility and performance apparently, https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles.

My habit is to use f-strings.

> The only other thing I'd say is should we let the user know through a message about the logging? We can couple this to that condition when the `logs` folder is created. Basically saying that from now on we will have logs so we can improve the usage of the terminal, and they can change the logging level allowed in the `config_terminal`.

This is just a feature just like anything else. We add it to the docs.

Reply:
TIL about lazy string formatting for logs